winterfell/config/mongoose.js

```diff
@@ -13,6 +18,7 @@ module.exports = function(environment) {
   });
   db.on('open', function () {
       console.log('Database connection opened at ' + address);
+      setupDb();
```

Comment:
we could have a clearDb() function that either drops the db or at least clears all entries.
Maybe a better option is to only add if it doesn't exist yet.